app/presenters/two_factor_options_presenter.rb

```diff
@@ -50,32 +50,32 @@ def show_security_level?
 
   def piv_cac_option
     return [] unless current_device_is_desktop?
-    [TwoFactorAuthentication::PivCacSelectionPresenter.new]
+    [TwoFactorAuthentication::PivCacSelectionPresenter.new(nil, @user)]
```

Comment:
I'd recommend using the attr accessor here (and in all these methods), because direct instance variable access will quietly return `nil` if typoed, but methods will throw errors

```suggestion
    [TwoFactorAuthentication::PivCacSelectionPresenter.new(nil, user)]
```